src/schemas.jl

```diff
@@ -110,13 +114,33 @@ function Base.showerror(io::IO, e::UnknownSchemaVersionError)
               This generally indicates that this schema has not been declared (i.e.
               the corresponding `@schema` and/or `@version` statements have not been
               executed) in the current Julia session.
+              """)
+    println(io)
+
+    if !ismissing(e.schema_provider_name)
+        provider_string = string(e.schema_provider_name)
+        if !ismissing(e.schema_provider_version)
+            provider_string *= string(" (version: ", e.schema_provider_version, ")")
```

Comment:
Having `Example (version: v1.0.0)` seems a bit verbose. Using `Example v1.0.0` seems fine to me:
```suggestion
            provider_string *= string(" ", e.schema_provider_version)
```